### PR TITLE
feat(i18n): Add french translations

### LINKED ISSUE #230: Add French translations
:fr:

@Antoine38660: @machour has asked to work on French translations, let us know if that's okay :)

### PR DESCRIPTION
**Pull request by machour**
Translations reviewed by @Antoine38660 
Closes #230

### CODE REVIEW

**Comment by lex111** [comment]
I would recommend yet not merge this into the master, because almost everything is ready to correct localization in #193 and there are a lot of changes that need to be considered.

**Comment by lex111** [comment]
To make the language appear in the settings, you need to add it to the [language-settings.js](https://github.com/gitpoint/git-point/blob/master/src/auth/screens/language-settings.js) file. Names of the language are written native, not in English.

**Comment by andrewda** [comment]
@lex111 Yeah, that sounds like a good plan!

**Comment by housseindjirdeh** [comment]
This was a lot of work and I can't thank you enough @machour for taking the time to do this. And thanks a million @Antoine38660 for collaborating and helping review. I really appreciate it :pray: 

Just some tiny things needed as @lex111 suggested (adding to `language-settings.js` and removing from `user-options.screen.js` and this should be good to merge :tada:

**Comment by machour** [comment]
This should now be fine :)

File: .all-contributorsrc [hunk 1]
```diff
@@ -238,6 +238,16 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "machour",
+      "name": "Mehdi Achour",
```
Comment (housseindjirdeh) [inline .all-contributorsrc]: 🎉

**Pull request merged**